modules/porous_flow/test/tests/sinks/s12.i

```diff
@@ -0,0 +1,150 @@
+# Test PorousFlowEnthalpySink boundary condition with specified pressure
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 3
+  nx = 2
+  ny = 2
+  nz = 2
+  xmin = 0
+  xmax = 10
+  ymin = 0
+  ymax = 10
+  zmin = 0
+  zmax = 10
+[]
+
+[GlobalParams]
+  PorousFlowDictator = dictator
+[]
+
+[UserObjects]
+  [dictator]
+    type = PorousFlowDictator
+    porous_flow_vars = 'pp temp'
+    number_fluid_phases = 1
+    number_fluid_components = 1
+  []
+  [pc]
+    type = PorousFlowCapillaryPressureConst
+    pc = 0.1
+  []
+[]
+
+[AuxVariables]
+  [p_aux]
+    initial_condition = 0.5
+  []
+[]
+
+[Variables]
+  [pp]
+    initial_condition = 1
+  []
+  [temp]
+    initial_condition = 2
+  []
+[]
+
+[Kernels]
+  [mass0]
+    type = PorousFlowMassTimeDerivative
+    fluid_component = 0
+    variable = pp
+  []
+
+  [heat_conduction]
+    type = TimeDerivative
+    variable = temp
+  []
+[]
+
+[Modules]
+  [FluidProperties]
+    [simple_fluid]
+      type = SimpleFluidProperties
+      bulk_modulus = 1
+      density0 = 10
+      thermal_expansion = 0
+      viscosity = 11
```

Comment:
Please remove `viscosity = 11` and replace by `cv = 1E-4`

Reply:
✅